src/fermilib/utils/_jellium.py

```diff
@@ -411,31 +461,29 @@ def jellium_model(n_dimensions, grid_length, length_scale,
         spinless: Bool, whether to use the spinless model or not.
         momentum_space: Boole, whether to return in momentum space (True)
             or position space (False).
+        energy_cutoff: Float, energy cutoff.
 
     Returns:
         hamiltonian: An instance of the FermionOperator class.
     """
     if momentum_space:
-        hamiltonian = momentum_kinetic_operator(n_dimensions,
-                                                grid_length,
-                                                length_scale,
-                                                spinless)
-        hamiltonian += momentum_potential_operator(n_dimensions,
-                                                   grid_length,
-                                                   length_scale,
-                                                   spinless)
+        hamiltonian = momentum_kinetic_operator(n_dimensions, grid_length,
+                                                length_scale, spinless,
+                                                energy_cutoff)
+        hamiltonian += momentum_potential_operator(n_dimensions, grid_length,
+                                                   length_scale, spinless,
+                                                   energy_cutoff)
     else:
-        hamiltonian = position_kinetic_operator(n_dimensions,
-                                                grid_length,
-                                                length_scale,
-                                                spinless)
-        hamiltonian += position_potential_operator(n_dimensions,
-                                                   grid_length,
-                                                   length_scale,
-                                                   spinless)
+        hamiltonian = position_kinetic_operator(n_dimensions, grid_length,
+                                                length_scale, spinless,
+                                                energy_cutoff)
+        hamiltonian += position_potential_operator(n_dimensions, grid_length,
+                                                   length_scale, spinless,
+                                                   energy_cutoff)
     return hamiltonian
 
 
+# TODO(wsws): Update with the latest formula, add U term, and add energy cut.
```

Comment:
I would normally be okay with merging changes that have this kind of TODO into develop but I think we might make a push to master tonight (Sunday) or tomorrow and I don't want to have this TODO end up in master. So I will either wait until we 1) push from develop to master or 2) you complete this TODO before merging this pull request. Does that make sense?